### PR TITLE
fix(docs): fix issue with template command example

### PR DESCRIPTION
**Pull request by caveman-dick**
The command in the docs was wrong, updated to reflect the actual command required

### CODE REVIEW

**Review by luhring — APPROVED** [review]
Great catch — thanks!

**Comment by spiffcs** [comment]
@caveman-dick if you can would you be able to sign your commit and we can merge this. Using GPG or S/MIME, you can sign tags and commits locally. These tags or commits are marked as verified on GitHub so other people can be confident that the changes come from a trusted source.

See [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) for more information.

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ Grype lets you define custom output formats, using [Go templates](https://golang
 
 - Grype's template processing uses the same data models as the `json` output format — so if you're wondering what data is available as you author a template, you can use the output from `grype <image> -o json` as a reference.
 
-**Example:** You could make Grype output data in CSV format by writing a Go template that renders CSV data and then running `grype <image> -o ~/path/to/csv.tmpl`.
+**Example:** You could make Grype output data in CSV format by writing a Go template that renders CSV data and then running `grype <image> -o template -t ~/path/to/csv.tmpl`.
```
Comment (Nicolentx32) [inline README.md:142]: Csv template

Comment (Nicolentx32) [reply README.md:142]: JSON

**Review by Nicolentx32 — APPROVED** [review]

**Comment by Nicolentx32** [comment]
> Great catch — thanks!

Y

**Pull request merged**